### PR TITLE
Docs/objects

### PR DESCRIPTION
**Pull request by joshLong145**
## Changes

- Item 1
- Item 2

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: src/highway/modules/objects.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Overview
+---
+
+# Objects
+Objects are data `blobs` persisted within an off chain storage network and given a content identifier, which repersents the object in the network, this objects is not kept on the `blockchain`. When uploading an object for storage, there must be validation against a [schema]() which must match in both property name, and type.
```
Comment (ntindle) [inline src/highway/modules/objects.md:6]: . This object

Comment (ntindle) [reply src/highway/modules/objects.md:6]: Link schema content. Outline should have the page already

File: src/highway/modules/objects.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+# Encoding.
+Objects are stored in the network as a `DAG`, which are made up of a series of data related to one another by `links` which allow content to be resolved into its complete form. This is possible though type assurance given to objects through the `schemas`.
```
Comment (ntindle) [inline src/highway/modules/objects.md:10]: What is a DAG

Comment (ntindle) [reply src/highway/modules/objects.md:10]: Which allow -> that allow

File: src/highway/modules/objects.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Overview
+---
+
+# Objects
+Objects are data `blobs` persisted within an off chain storage network and given a content identifier, which repersents the object in the network, this objects is not kept on the `blockchain`. When uploading an object for storage, there must be validation against a [schema]() which must match in both property name, and type.
+
+
+# Encoding.
```
Comment (ntindle) [inline src/highway/modules/objects.md:9]: No period

File: src/highway/modules/objects.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+Objects are stored in the network as a `DAG`, which are made up of a series of data related to one another by `links` which allow content to be resolved into its complete form. This is possible though type assurance given to objects through the `schemas`.
+
+# Referencing
+ It should be noted that once an object is stored. its `cid` must be associated with a [buckets]() to be retrievable. Buckets allow objects to be searched upon by schema, and meta-data. Currently only searching by a schema is supported, but indexing of buckets is expected soon.
```
Comment (ntindle) [inline src/highway/modules/objects.md:13]: stored,

Comment (ntindle) [reply src/highway/modules/objects.md:13]: Link buckets

Comment (ntindle) [reply src/highway/modules/objects.md:13]: Put a date in parentheses on currently

File: src/highway/modules/objects.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+ # Examples
+
+See [Speedway]() Documentation for information on how to upload content through a managed Content enviorment or our `CLI`.
```
Comment (ntindle) [inline src/highway/modules/objects.md:17]: Link speedway

File: src/highway/modules/objects.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+# Encoding.
+Objects are stored in the network as a `DAG`, which are made up of a series of data related to one another by `links` which allow content to be resolved into its complete form. This is possible though type assurance given to objects through the `schemas`.
+
+# Referencing
+ It should be noted that once an object is stored. its `cid` must be associated with a [buckets]() to be retrievable. Buckets allow objects to be searched upon by schema, and meta-data. Currently only searching by a schema is supported, but indexing of buckets is expected soon.
+
+ # Examples
+
+See [Speedway]() Documentation for information on how to upload content through a managed Content enviorment or our `CLI`. 
+See [Motor-SDK]() Documentation for how to interface with objects through there for framework interaction with the object sore.
```
Comment (ntindle) [inline src/highway/modules/objects.md:18]: Link motor

**Review by ntindle — CHANGES_REQUESTED** [review]

File: src/highway/modules/objects.md (hunk 1 above)
Comment (joshLong145) [reply src/highway/modules/objects.md:6]: > Link schema content. Outline should have the page already

i will be adding links soon.

File: src/highway/modules/objects.md (hunk 2 above)
Comment (ma-sonr) [reply src/highway/modules/objects.md:10]: Directed acyclic graph , but the way its explained is confusing: could you add a diagram?

File: src/highway/modules/objects.md (hunk 3 above)
Comment (ma-sonr) [reply src/highway/modules/objects.md:9]: A very basic example would be useful

**Pull request closed without merging**